src/client/components/CreateSeason.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react';
+
+import { TitleForm } from './seasonform/TitleForm';
+import { CompetitorForm } from './seasonform/CompetitorForm';
+import { TeamForm } from './seasonform/TeamForm';
+import { ParticipantForm } from './seasonform/ParticipantForm';
+import { MappingForm } from './seasonform/MappingForm';
+
+export const CreateSeason = () => {
```

Comment:
you do not import/use the user prop you pass down from app.jsx here?

Reply:
Must have been a leftover typo from a copy/paste in app, ill get it removed